3rdparty/mbedtls/CMakeLists.txt

```diff
@@ -73,6 +83,11 @@ ExternalProject_Add(mbedtls-wrap
 ExternalProject_Get_property(mbedtls-wrap SOURCE_DIR)
 ExternalProject_Get_property(mbedtls-wrap BINARY_DIR)
 
+# Add the mbedtls_hardware_poll function to the mbedcrypto library.
+add_custom_command(TARGET mbedtls-wrap
+    POST_BUILD
+    COMMAND ${CMAKE_AR} r ${BINARY_DIR}/library/libmbedcrypto.a $<TARGET_OBJECTS:oe_mbedtls_hardware_poll>)
```

Comment:
Can this be ${LLVM_AR}?

Reply:
`${LLVM_AR}` is the path to the llvm archiver on clang on windows.  See the code that line 27:
```
  find_program(LLVM_AR "llvm-ar.exe")
  set(MBEDTLS_LINKER "-DCMAKE_AR=${LLVM_AR}")
  set(MBEDTLS_LINKER "-DCMAKE_AR=${LLVM_AR}")
  set(MBEDTLS_COMPILER "")
  set(MBEDTLS_COMPILER "")
  set(CMAKE_AR ${LLVM_AR})
```
So, `${CMAKE_AR} == ${LLVM_AR}` on clang-on-windows and the correct archiver in other systems.